googleanalytics/googleanalytics-connector/googleanalytics-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/connector.xml

```diff
@@ -32,7 +32,15 @@
       <dependency component="googleanalytics-accounts"/>
       <dependency component="googleanalytics-config"/>
       <dependency component="googleanalytics-multiChannelFunnelsData"/>
-      <description>WSO2 ESB Connector Library for GoogleAnalytics API.</description>
+      <dependency component="googleanalytics-views"/>
+      <dependency component="googleanalytics-goals"/>
+      <dependency component="googleanalytics-uploads"/>
+      <dependency component="googleanalytics-webProperties"/>
+      <dependency component="googleanalytics-webPropertyUserLink"/>
+      <dependency component="googleanalytics-profileUserLink"/>
+      <dependency component="googleanalytics-profileFilterLink"/>
+
+       <description>WSO2 ESB Connector Library for GoogleAnalytics API.</description>
```

Comment:
Fixed in the new repo https://github.com/wso2-extensions/esb-connector-googleanalytics